apps/editing-toolkit/editing-toolkit-plugin/block-inserter-modifications/index.php

```diff
@@ -55,6 +55,7 @@ function enqueue_style( $filename ) {
 		array(),
 		filemtime( plugin_dir_path( __FILE__ ) . 'dist/' . $style_file )
 	);
+	wp_set_script_translations( $filename, 'full-site-editing' );
```

Comment:
Is there a reason to have this in `enqueue_style` and not in `enqueue_script`?

Reply:
Thanks a lot for catching this, fixed.